### DISCUSSION TITLE
🧩 FEATURE REQUEST:  all squad activities for slack integration

### DISCUSSION THREAD

**Discussion by warengonzaga** [dailydotdev/daily#1564] [Feature Requests]
### Concise Description

Add support for activities including upvotes, mention, and comments for slack integration

### Description

Hey there, Waren here from thirdweb community team. I would like to request to add support for all kinds of activities from our Squad in dailydev to Slack integration. This will help us monitor the Squad activities such as upvotes, mentions, and comments.

### Solution Proposed

Add notifications for upvotes, comments, and mentions. It is even better, a notification when someone joins the squad.

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @warengonzaga,

That's a great suggestion. We will consider it

**Comment by warengonzaga** [comment]
Thank you let me know once available! Willing to test it.

**Comment by PredictiveManish** [comment]
Is someone working on this feature??

**Comment by idoshamun** [comment]
@PredictiveManish nope. it's currently not prioritized

**Reply by PredictiveManish** [reply]
> @PredictiveManish nope. it's currently not prioritized

Ok @idoshamun
